WootricSDK/WootricSDKTests/SEGWootricTests.m

```diff
@@ -23,12 +23,12 @@
 // THE SOFTWARE.
```

Comment:
So just to clarify, the changes here do not make EBS available to our iOS SDK installs?

Reply:
This does not make it available to Segment installations yet, that'll come in another PR.